en/advanced_features/precland.md

```diff
@@ -1,9 +1,13 @@
 # Precision Landing
 
-PX4 supports precision landing for *multicopters* (from PX4 v1.7.4) using the [IR-LOCK Sensor](https://irlock.com/products/ir-lock-sensor-precision-landing-kit), an IR beacon (e.g. [IR-LOCK MarkOne](https://irlock.com/collections/markone)), and a downward facing range sensor.
+PX4 supports precision landing for *multicopters* using a landing beacon/target.
```

Comment:
Perhaps " using a landing beacon/target." should be " using a landing target (beacon, visual marker, or any other positioning mechanism)."?